tools/tm-bench/main.go

```diff
@@ -93,6 +95,19 @@ Examples:
 		"broadcast_tx_"+broadcastTxMethod,
 	)
 
+	//catch Interrupt and quit tm-bench
```

Comment:
I don't think this comment is needed

Reply:
Should be removed or have proper form of: `Catch interrupt and quit.`